src/main/java/org/zaproxy/admin/GenerateAddonsYAML.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2015 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.admin;
+
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.nio.file.Files;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * Command line tool for generating the addons markdown file for website.
+ *
+ * @author nirojan
+ */
+public class GenerateAddonsYAML {
+
+    private static final String ZAP_VERSIONS_FILE_NAME = "ZapVersions-2.9.xml";
+    private static final String NEXT_LINE = "\n";
+    private static final String EMPTY_STRING = "";
+    private static final String OUTPUT_DIR = "addons.yml";
+
+
+    public static void main(String[] args) throws Exception {
+
+        StringBuilder sb = new StringBuilder("---" + NEXT_LINE);
+        File xmlFile = new File(ZAP_VERSIONS_FILE_NAME);
+        if(xmlFile.exists()){
+
+            DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
```

Comment:
It can be used the  ZAP class `AddOnCollection` instead of manually parsing the ZapVersions file.

Reply:
Hi @thc202  thanks for the pointer, can I know how can I get the date from AddOn class?
![image](https://user-images.githubusercontent.com/13045528/74641571-d383b580-5171-11ea-8c06-a6d5080e0123.png)

Reply:
Seems we are not exposing that :/ It's read in `ZapVersionsAddOnEntry` but never used.

Reply:
Okay, then shall I modify the class to expose it?

Reply:
Yeah, we should expose that, although it would not help here until we release a new version (you can use `ZapVersionsAddOnEntry` directly or keep as is).